src/base/utils/net.h

```diff
@@ -0,0 +1,13 @@
+#ifndef BASE_UTILS_NET_H
+#define BASE_UTILS_NET_H
+#include <QString>
```

Comment:
Fixed, but I think it is kind of a dirty hack and premature optimization (which is a root of all evil (c) ).